lib/net/pool.js

```diff
@@ -490,6 +531,7 @@ class Pool extends EventEmitter {
    * @returns {Promise}
    */
 
+  // TODO: does this need brontide solution?
```

Comment:
Now that I think about it, this doesn't need any work to accommodate for brontide unless we were using bolt10 because the identity keys would not fit in `A` or `AAAA` records. If we did write a bolt10 server/client, this is where it could be used.